### DISCUSSION TITLE
Help with package - Wing101

### DISCUSSION THREAD

**Discussion by danielsherratt** [ScoopInstaller/Scoop#5241] [General]
Hi, I need some help with a package im trying to create

Heres my JSON:

{
    "version": "9.0.0.6",
    "description": "Wing 101 is a very simple free Python IDE designed for teaching beginning programmers.",
    "homepage": "https://wingware.com/downloads/wing-101",
    "license": "Freeware",
    "architecture": {
        "64bit": {
            "url": "https://wingware.com/pub/wing-101/9.0.0.6/wing-101-9.0.0.6.exe",
            "hash": "04c29005aa9fe7f68af77b911171e8b2b8b70beda1114ad65548287aed756a29"
        }
    },
	"installer": {
		"file": "wing-101-9.0.0.6.exe",
		"args": "/silent"
	},
	"uninstaller": {
		"file": "c:\\program files\\wing 101 9\\unins000.exe"
	},
	"checkver": {
        "url": "https://wingware.com/downloads/wing-101",
        "regex": "Download Wing 101 version ([\\d\\.]{7,11})"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://wingware.com/pub/wing-101/$version/wing-101-$version.exe"
            }
		}
	}
}

The package installs fine, but when I try to uninstall it, it states the package cant be found - \scoop\apps\wing101\9.0.0.6\c:\program files\wing 101 9\unins000.exe is missing, skipping.
Which I believe is the correct response, whats the best way to get out of scoop? also I've noticed the unins changes sometimes from unins000 to unins001

I tried installing version 9.0.0.4 which installed fine, but then trying scoop update wing101 it said I had the latest version, I thought it would check wingware/downloads/wing-101 for the regex and return the new value 9.0.0.6, then autoupdate using that as $version

Any help is greatly appreciated

Daniel

**Comment by niheaven** [comment]
Please refer https://github.com/ScoopInstaller/Extras/pull/9677
